cli/internal/runsummary/run_summary.go

```diff
@@ -16,7 +25,19 @@ type RunSummary struct {
 	TurboVersion      string             `json:"turboVersion"`
 	GlobalHashSummary *GlobalHashSummary `json:"globalHashSummary"`
 	Packages          []string           `json:"packages"`
-	Tasks             []TaskSummary      `json:"tasks"`
+	Tasks             *[]*TaskSummary    `json:"tasks"`
+}
+
+// Save saves the run summary to a file
+func (summary *RunSummary) Save(singlePackage bool) error {
+	json, err := summary.FormatJSON(singlePackage)
+	if err != nil {
+		return err
+	}
+
+	nonce := "abcdef"
+	filename := fmt.Sprintf("vercel-turbo-run-%s-%s.json", nonce, time.Now().Format(time.RFC3339))
```

Comment:
this should _probably_ always go in the root of the directory. e.g. if we're running turbo from a workspace, we need to handle that case.